translations/admin.en.json

```diff
@@ -10,5 +10,8 @@
     "app.album_selection_label": "{count} {count, plural, =1 {album} other {albums}} selected",
     "app.select_albums": "Select albums",
     "app.no_album_selected": "No album selected",
-    "app.select_album": "Select album"
+    "app.select_album": "Select album",
+    "sulu_activity.description.albums.created": "{userFullName} has created the album \"{resourceTitle}\"",
+    "sulu_activity.description.albums.modified": "{userFullName} has changed the album \"{resourceTitle}\"",
+    "sulu_activity.description.albums.removed": "{userFullName} has removed the album \"{resourceTitle}\""
```

Comment:
These translation keys consist of `sulu_activity.description` as fixed part, the `resourceKey` (`albums`) and the `eventType` (`removed`).

The available translation parameters can be found [here](https://github.com/sulu/sulu/blob/5e0db29766f99ae62528123a5ca78200bbc21be0/src/Sulu/Bundle/ActivityBundle/UserInterface/Controller/ActivityController.php#L426-L449)

To use variables of the `context`, you have to use `context_myVar`.